rustls/src/key_log.rs

```diff
@@ -10,7 +12,7 @@
 ///
 /// See [`KeyLogFile`](crate::KeyLogFile) that implements the standard
 /// `SSLKEYLOGFILE` environment variable behaviour.
-pub trait KeyLog: Send + Sync {
+pub trait KeyLog: Send + Sync + Debug {
```

Comment:
Suggestion: I like to order bounds in order of least common to most common, so I'd probably put `Debug` before `Send` and `Sync`.

Reply:
Fixed throughout